zng/docs/spec.md

```diff
@@ -616,6 +616,10 @@ of a value:
 In addition, `-` must be escaped if representing the single ASCII byte equal
 to `-` as opposed to representing an unset value.
 
+> `\x` followed by anything other than two hexadecimal digits is not a valid
+> escape sequence. The behavior of an implementation that encounters such
+> invalid sequences in a `bstring` or `string` type is undefined.
```

Comment:
\x not allowed in string

Reply:
@mccanne: Thanks for catching that. Part of why I made that mistake was because I was mimicking some existing text just above that in the same section that spoke of:

> These special characters must be hex escaped if they appear within a `bstring` or a `string` type:

On the assumption that's wrong too, I've pushed another commit to fix both references. Once I hear you're 👍 with that change I'll go ahead and merge.

Reply:
@aswan chimed in with an approval since I made the additional commit referenced in the comment above, so I'm gonna go ahead and merge. We can always tweak it more.